nginx_ingress_controller/datadog_checks/nginx_ingress_controller/nginx_ingress_controller.py

```diff
@@ -35,6 +35,7 @@ def __init__(self, name, init_config, instances):
                         # controller metrics
                         {'nginx_ingress_controller_success': 'controller.reload.success'},
                         {'nginx_ingress_controller_ingress_upstream_latency_seconds': 'controller.upstream.latency'},
+                        {'nginx_ingress_controller_response_duration_seconds': 'controller.response.duration'},
```

Comment:
Can we also assert this metric is collected here https://github.com/DataDog/integrations-core/blob/7d53adadc873c6450025f4cf036f1c8d73296e04/nginx_ingress_controller/tests/test_nginx_ingress_controller.py#L31

Reply:
Added those assertions in the file, thanks for pointing me to the file. :)